model/cat/indices/cat_indicies/operations.smithy

```diff
@@ -7,6 +7,8 @@
 $version: "2"
 namespace OpenSearch
 
+use opensearch.openapi#openApiVendorExtensions
```

Comment:
@nhtruong sorry, just want to hear your thoughts since Smithy is not my thing, BUT wouldn't `use opensearch.openapi#` already imply that those are vendor extensions for OpenAPI? 

```
use opensearch.openapi#vendorExtensions
```

That would be pretty clean, no?